src/Controller/UI/ReportResearchGroupDatasetStatusController.php

```diff
@@ -175,19 +257,28 @@ protected function getData(array $options): array
     /**
      * Create a CSV download filename that contains the truncated research group name and the date/timeto.
      *
-     * @param string $researchGroupName The name of the Research Group which is the subject of the report.
-     * @param string $researchGroupId   The ID of the Research Group which is the subject of the report.
+     * @param string  $researchGroupName The name of the Research Group which is the subject of the report.
+     * @param string  $researchGroupId   The ID of the Research Group which is the subject of the report.
+     * @param integer $version           Report version number.
      *
      * @return string
      */
-    private function createCsvReportFileName(string $researchGroupName, string $researchGroupId)
+    private function createCsvReportFileName(string $researchGroupName, string $researchGroupId, int $version)
     {
         $nowDateTimeString = date(self::REPORTFILENAMEDATETIMEFORMAT);
-        $researchGroupNameSubstring = substr($researchGroupName, 0, self::MAXRESEARCHGROUPLENGTH);
-        $tempFileName = $researchGroupNameSubstring . '_' . $researchGroupId
-            . '_'
+
+        if ($version === self::REPORT_VERSION_THREE) {
+            $tempFileName = $researchGroupName . '_'
             . $nowDateTimeString
```

Comment:
PO wanted a yyyy-mm-dd HH:mm format, this results in a dd-mm-yyyy HH:mm format. Check with PO if this is ok as is though.

Reply:
No, i think it results in Y-m-d.

Reply:
This line is for the filename timestamp, i get it you were talking about timestamp in one of the date fields and you were right, it is not in the right format. will change it